js/src/blocks/product-select-with-text-field/block.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "google-listings-and-ads/product-select-with-text-field",
+	"title": "Product select control with text control",
```

Comment:
💅  In line with the other block. WDYT to keep is as just Fields? 

"Product Select with Text Field"

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/commit/776ffb843618a06e08a985a84f56474ed3f49283.